src/ParachainStaking.sol

```diff
@@ -214,7 +214,7 @@ contract ParachainStaking is Parachain {
         emit StakeWithdrawn(msg.sender);
         emit ParachainStakeWithdrawn(_paraId, msg.sender);
 
-        reportStakeWithdrawn(parachain, msg.sender, _amount);
+        reportStakeWithdrawn(parachain, abi.encodePacked(msg.sender), _amount);
```

Comment:
This doesnt seem correct, it should be the account id of the reporter on consumer chain rather than just a bytes encoding of the msg.sender address. The reportSlash call seems to handle this correctly.

Reply:
Thanks for spotting. Fixed in latest commit.